README.md

```diff
@@ -1,168 +1,170 @@
+**Note!** Plugin settings can have any other fields which needs for plugin (not only special fields `excludePaths` and `techs`). Set of the fields is defined by implementation of plugin.
 
-## Написание плагинов
+## How to create your own plugin
 
-JS-файл следующего формата:
+You need to create the JavaScript file in following format:
 
 ```js
 module.exports = {
     /**
-     * Предопределенный конфиг для плагина,
-     * данный конфиг будет объединен с пользовательским конфигом
+     * Default plugin settings (it will be merged with settings from configuration file)
      * @returns {Object}
      */
     configure: function() {
         return {
-            // конфиг
+            // settings
         }
     },
 
     /**
+     * Checks which needs the information about all BEM entities of the project
      * @param {Entity[]} entities
      * @param {Config} config
      */
     forEntities: function(entities, config) {
-        // Использование этой функции будет полезно,
-        // если для реализации проверки необходимо знание
-        // о всех БЭМ-сущностях проекта
+        ...
     },
 
     /**
+     * Checks of the specific BEM entity
      * @param {Entity} entity
      * @param {Config} config
      */
     forEachEntity: function(entity, config) {
-        // Использование этой функции будет полезно,
-        // если для реализации проверки достаточно знание
-        // о каждой из БЭМ-сущностей в отдельности
+        ...
     },
 
     /**
+     * Checks of separate technology of the specific BEM entity
      * @param {Object} tech
      * @param {Entity} entity
      * @param {Config} config
      */
     forEachTech: function(tech, entity, config) {
-        // Использование этой функции будет полезно,
-        // если для реализации проверки достаточно знание
-        // о каждой из технологий БЭМ-сущностей в отдельности
+        ...
     }
 };
 ```
 
-**Замечание!** Реализуемый плагин может содержать как одну из функций `forEntities`, `forEachEntity`, `forEachTech`, так и все три, при этом функция `configure` не является обязательной.
+**Note!** Your plugin should contains at least one of functions `forEntities`, `forEachEntity`, `forEachTech`, but `configure` function is not required.
 
 #### Entity
 
-БЭМ-сущность проверяемого проекта (блок, элемент, модификатор).
+BEM entity (block, element or modifier).
 
 **Entity.prototype.getTechs**<br>
-**@returns** *{Tech[]}* - список технологий, в которых реализована данная БЭМ-сущность
+**@returns** *{Tech[]}* - list of technologies, in which implemented this BEM entity
 
 **Entity.prototype.getTechByName**<br>
-**@param** *{String}* – имя технологии (css, js и т.д.)<br>
-**@returns** *{Tech}* – технология БЭМ-сущности
+**@param** *{String}* – technology name (css, js etc. - all that located after the first dot in file name)<br>
```

Comment:
in **a** file

that –> which

----

substring which goes after the firs dot in a file name